### ISSUE TITLE
[REQUEST] Module acquisition choice after rare metal collection

### ISSUE THREAD

**Issue by ebsmarch** [Liam-Harrison/Celeritas#219]
After collecting 100 rare metals (which are dropped from asteroids or destroyed enemy ships), the player has the option to go into the build menu and build a randomly selected module out of a choice of 3. This interface will be present on the build menu.

When entering the build menu, there will be a popup, tab, or button that flashes or makes itself very obvious to the player stating they can now craft a new module. When they click there, they are given a selection of 3 modules. The modules' shape, size, and full detail of effects will be listed, and it must be made clear that only one is selectable. There must also be a counter of how many rare metals they have currently, and a notice that making this module will consume 100 pieces of rare metal.  They can click on a module to select it, but to confirm their choice, there should be a button that locks it in.

To summarise, the interface must be clear, and have the following information:

> The modules as their shapes, with information underneath or next to them showing their effects and title/name at all times

> A counter of the rare metals, and it must be clear that making one of these modules will consume 100 rare metals (this can be a -100 underneath the counter that flashes when the player hovers over a module option)

> A confirm button that lights up and is selectable once a module option has been selected, the module selected can be highlighted or coloured so their choice is clear
